src/style.css

```diff
@@ -0,0 +1,36 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --star-size: 1rem;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars :first-child {
+  margin-left: 0;
+}
+
+.stars__star,
+.stars__star--active {
+  background-position: center;
+  background-repeat: no-repeat;
+  width: var(--star-size);
+  height: var(--star-size);
+  margin-left: 0.25rem;
+}
+
+.stars__star {
+  background-image: url("/images/star.svg");
+}
+
+.stars__star--active {
+  background-image: url("/images/star-active.svg");
+}
```

Comment:
You need to make some changes in here.

1.  lets set `background-image: url("/images/star.svg");` in `.stars__star` selector.

2. Get rid of class with modificator both from html and css, use only `stars__star` class.

3.  Determine which star is supposed to have `background-image: url("/images/star-active.svg");` with help of :nth-child and stars--N - here are [MDN docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child)